examples/gnn_explainer_link_pred.py

```diff
@@ -0,0 +1,126 @@
+import os.path as osp
+
+import torch
+import torch.nn.functional as F
+from sklearn.metrics import roc_auc_score
+
+import torch_geometric.transforms as T
+from torch_geometric.datasets import Planetoid
+from torch_geometric.explain import (
+    Explainer,
+    ExplainerConfig,
+    GNNExplainer,
+    ModelConfig,
+)
+from torch_geometric.nn import GCNConv
+from torch_geometric.utils import negative_sampling
+
+dataset = 'Cora'
+path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data', 'Planetoid')
+device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+transform = T.Compose([
+    T.NormalizeFeatures(),
+    T.ToDevice(device),
+    T.RandomLinkSplit(num_val=0.05, num_test=0.1, is_undirected=True,
+                      add_negative_train_samples=False),
```

Comment:
```suggestion
    T.RandomLinkSplit(num_val=0.05, num_test=0.1, is_undirected=True),
```
Can we make this example as minimal as possible? I don't think we desperately need the logic of on-the-fly negative sampling in this example.